modules/era/sql/era_droplist.sql

```diff
@@ -15811,22 +15791,22 @@ INSERT INTO `mob_droplist` VALUES (1935,2,0,1000,748,0);          -- Gold Beastc
 
 -- ZoneID: 141 - Orcish Serjeant
 INSERT INTO `mob_droplist` VALUES (1936,0,0,1000,530,@RARE);    -- Copy Of The Castle Floor Plans (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1936,0,0,1000,1024,@RARE);   -- Ghelsba Chest Key (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1936,0,0,1000,1024,@VRARE);  -- Ghelsba Chest Key (Very Rare, 1%)
```

Comment:
http://www.ffxidb.com/zones/141/orcish-serjeant

5% looks to be like the correct value for this

Reply:
Bumped up to 5%